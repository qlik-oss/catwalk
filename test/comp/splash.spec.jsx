import React from 'react';
import renderer from 'react-test-renderer';
import Splash from '../../src/components/splash';

describe('Splash', () => {
  it('should render docList correctly', () => {
    const docs = [{
      qDocId: '/data/Consumer_Sales.qvf', qMeta: { description: 'See through the eyes of a consumer goods company.  Analyze sales data by sales rep, by region, by product, etc.' }, qTitle: 'Consumer_Sales',
    }, {
      qDocId: '/data/drugcases.qvf', qMeta: { description: 'This Dashboard is based on the 2015 Q4 data.' }, qTitle: 'FDA - Drug Cases',
    }, {
      qDocId: '/data/New_QVNCycles_with_measures_dimensions_visualizations_.qvf', qMeta: { description: 'An app with some measures and dimensions and master visualizations.\nTexten finns också på skånska.' }, qTitle: 'New_QVNCycles_with_measures_dimensions_visualizations_',
    }];
    const error = null;
    const engineURL = '';

    const tree = renderer.create(<Splash
      docs={docs}
      error={error}
      engineURL={engineURL}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should display message if empty docList array', () => {
    const docs = [];
    const error = null;
    const engineURL = '';

    const tree = renderer.create(<Splash
      docs={docs}
      error={error}
      engineURL={engineURL}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
