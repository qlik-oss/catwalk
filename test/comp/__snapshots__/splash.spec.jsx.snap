// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'should display message if empty docList array' 1`] = `
Array [
  <div
    className="center-content"
  >
    <div
      className="splash"
    >
      <span
        className="SVGInline logo"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<svg class=\\"SVGInline-svg logo-svg\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 481.91 121.64\\"><path d=\\"M450.34 70.31l29.61-34.76h-31.56l-19.84 24.68-2.66 3.52V0h-26.33v120.08h26.33V96.02l7.42-7.43 18.44 31.49h30.16l-31.57-49.77zM368.75.08h26.41v120h-26.41zM364.81 63.52q-.23-13.75-9.61-21.65t-25.55-7.89q-16.17 0-26.6 7.65t-10.43 19.9h26.33q0-9.26 9.38-9.26 10.08 0 10.08 11.7v4.48h-8.12q-19.38 0-29.53 7T290.6 96.23a23 23 0 0 0 8.35 18.14 30 30 0 0 0 20.31 7.23q13 0 20.31-9.14a29.69 29.69 0 0 0 2.5 7.58h26.33v-1.33q-3.44-6.17-3.59-17.81V63.52zm-26.41 32.5a13.23 13.23 0 0 1-4.65 4.77 14.69 14.69 0 0 1-8.16 2.11 8.76 8.76 0 0 1-6.17-2.23 7.53 7.53 0 0 1-2.42-5.82q0-12.34 13.75-12.34h7.66v13.52z\\"/><path d=\\"M271.81 35.55l-8.51 49.82-13.28-49.84h-18.36l-13.44 51-8.36-51h-31.83v-21h-6.59c1.31 3.11-5.47 7.1-6.24 9.71-1.47 4.92 2.32 11 1.15 16.33-1.41 6.42-5.85 10.52-11 14.52-.89.7-2.26 1.67-3.69 2.8v18.67c1.36 4.54.92 7.53 5.25 11.43 2.24 2 6 3.84 7.88 5.88a22.93 22.93 0 0 1 3.4 4.64c2.37 5.2 7.2 12.79 6.64 18 .06-.53 0 .13 0 0-.29 3.21-3.79 5.49-6.39 3-1.41-1.36-2-6.24-3.24-8-5.13-7.52-9.18-10.18-13.32-11.49.61 6.58 2.69 11.67 6.27 15.21q6.48 6.41 20.08 6.41a48.61 48.61 0 0 0 14.88-2.27v-18.82a38.63 38.63 0 0 1-6.56.47q-5.08 0-6.8-1.87t-1.72-6.64V53.83h10.9l15.61 66.25h22.27l14.06-50.16 14 50.16h22.27l19.92-84.53h-25.25z\\"/><path d=\\"M158.67 37.82c-.24-6.51-3.56-12.68.53-18.95a18.38 18.38 0 0 1 4.51-4.34h-12v21h-10.79v13.08a26.07 26.07 0 0 0-5.72-6.74q-9.37-7.89-25.55-7.89t-26.6 7.65q-7.6 5.57-9.66 13.59a29.4 29.4 0 0 0-7.75-12.1q-9.69-9.14-26.17-9.14-12.19 0-21.09 5.27A34.41 34.41 0 0 0 4.73 54.37 51.48 51.48 0 0 0 0 76.99v1.09q0 20.47 10.63 32a34 34 0 0 0 7.65 6.24c1.17-1.06 3.55-1.31 4.94-2.06 2.56-1.36 4.74-3.85 6.91-5.69 4.26-3.63 10.12-7.41 12.36-12.5.72-1.63 3.83-10.57 2.72-12.46-2-3.48-8.2-1.59-10.52-1.64-3.87-.08-6.76-.39-9.49-2.72a2 2 0 0 1-.65-1.94c.67-3.15 1.52-7 3.69-9.49 1-1.12 2.84-1.92 3.28-3.38.51-1.66-.94-3.07-1-4.72a.19.19 0 0 1 .28-.16c.7.41 2.26 1.16 3.81 2-6.27-7.84 7.59.45 7.85.5 5.46 1 12.9 2.19 18.37 1.14 3.12-.6 5.73-2 9-2.05 3.71-.05 7.29 1.3 11 1 6.92-.49 13.7-2.72 20.69-3.23 7.55-.55 14.71.58 22.22.83 7.35.25 13.15-3.45 19.15-7a65.83 65.83 0 0 1 6.77-3.56c5.16-2.21 9.22-5.67 9.01-11.37zM75.02 100.06c-5-3.21-7.4-2.07-12.38-.82-2.24.56-4.53.61-6.65 1.55-4.42 2-7.62 5.86-11.85 8.16a105.74 105.74 0 0 0-12.67 8.19 29.86 29.86 0 0 1-4.81 2.78 46.9 46.9 0 0 0 13 1.72q15.7 0 25.66-8.91a29.76 29.76 0 0 0 6.57-8.3 23.52 23.52 0 0 0 7 10 29.64 29.64 0 0 0 3.85 2.8 1.4 1.4 0 0 0 .78-.6c.23-.6-.92-2.94-1-3.62-.45-4.9-3.19-10.16-7.5-12.95zM144.8 98.47a41.22 41.22 0 0 1-4.81-1.22c-1 3.11-.65 4.93.44 7.84.7 1.86 2.23 3 1.71 5.12-.76 3.16-5.38 7.19-9.81 9.86h16.06v-1.33q-3.44-6.17-3.59-17.81v-2.46zM120.65 101.76c-8.29-6.43-22.64-4-32.57-6.45-2 6.72 6.38 18.89 1.6 25a33.38 33.38 0 0 0 9.56 1.35q13 0 20.31-9.14a40.36 40.36 0 0 0 1.49 5.3 5.51 5.51 0 0 0 1-.3 20.29 20.29 0 0 0 5.92-4.73c2.84-4.04-3.85-8.34-7.31-11.03z\\"/></svg>",
          }
        }
      />
      <h3>
        Sets the stage for your Qlik data models
      </h3>
      <div>
        <p>
          WebSocket connected but no open app, in addition the app list was empty.
        </p>
        <p>
          Please make sure there are apps accessible on this engine and reload the page, or connect to another one:
        </p>
        <form
          onSubmit={[Function]}
        >
          <label
            htmlFor="engineURL"
          >
            <input
              defaultValue=""
              id="engineURL"
              type="text"
            />
          </label>
          <input
            type="submit"
            value="Connect"
          />
        </form>
      </div>
    </div>
  </div>,
  <div
    className="cat-with-bubble"
    style={null}
  >
    <span
      className="SVGInline close-me"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg class=\\"SVGInline-svg close-me-svg\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\"><g data-name=\\"Layer 2\\"><g data-name=\\"close\\"><rect width=\\"24\\" height=\\"24\\" transform=\\"rotate(180 12 12)\\" opacity=\\"0\\"/><path d=\\"M13.41 12l4.3-4.29a1 1 0 1 0-1.42-1.42L12 10.59l-4.29-4.3a1 1 0 0 0-1.42 1.42l4.3 4.29-4.3 4.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l4.29-4.3 4.29 4.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z\\"/></g></g></svg>",
        }
      }
      onClick={[Function]}
      title="Hide cat"
    />
    <span
      className="SVGInline cat"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg class=\\"SVGInline-svg cat-svg\\" version=\\"1.1\\" id=\\"Layer_1\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" x=\\"0px\\" y=\\"0px\\"
	 viewBox=\\"0 0 864 864\\" style=\\"enable-background:new 0 0 864 864;\\" xml:space=\\"preserve\\">
<style type=\\"text/css\\">
	.st0{fill:none;stroke:#000000;stroke-width:2;stroke-linecap:round;stroke-miterlimit:10;}
</style>
<g>
	<path d=\\"M677.9,236.7c-5.9-2.7-12.3-3.5-18.7-4.3c-8.4-1-16.8-1.4-25.2-0.9c-14.9,0.8-29.5,3.5-43.9,7.2s-28.8,7.9-42.7,13.3
		c-10.8,4.2-22.2,6.1-33.5,8.2c-11.1,2.1-22.3,3.2-33.4,4.4c-11.4,1.1-22.5-0.3-32.5-6.4c-2.7-1.6-5.6-2.8-8.7-3.7
		c-11.3-3.2-21.4-8.8-31.6-14.2c-12.4-6.5-24.7-13.2-37.2-19.5c-12.5-6.4-25.1-12.5-37.9-18.3c-12.5-5.7-23.4-13.9-33.9-22.7
		c-7.5-6.4-14.9-12.7-23.4-17.7c-9.5-5.6-19.6-9.3-30.6-10.8c-7.8-1.1-13.9-5.2-18.6-11.6c-6.5-8.9-11.5-18.6-15.6-28.8
		c-4.4-10.9-8.5-22-14.1-32.4c-7.4-13.8-15.7-27-26.7-38.5c-8.8-8.9-19.2-15.2-31.4-18.2c-3.3-0.8-6.6-1.8-9.9-2.7
		c-2.7,0-5.5,0-8.2,0c-4.2,1.3-7.5,4-10.3,7.3c-2.9,3.4-6.5,6.2-9.3,9.6c-1.8,2.2-3.3,4.5-4.4,7C90.3,57,82.1,69.8,73,82.2
		c-5.5,7.6-10.3,15.5-14.6,23.7c-4.3,8.2-7.9,16.7-11.3,25.4c-2.3,6-2.8,12.5-4,18.7c-0.9,4.3-2.4,8.3-4.4,12.2
		c-2.7,5.3-4.4,11-7.4,16.2c-3.7,6.2-7.6,12.1-12.8,17.1c-1.7,1.5-13.7,10.7-13.7,11.5c0,12,0,23.9,0,35.9c0,12,0,23.9,0,35.9
		c0,12-0.1,23.9-0.1,35.9s0,23.9,0,35.9s0.1,23.9,0.1,35.9c0,12,0,23.9,0,35.9s-0.1,23.9-0.1,35.9s0,23.9,0,35.9c0,12,0,23.9,0,35.9
		s0,23.9,0,35.9c0,12,0.1,23.9,0.1,35.9s-0.1,23.9-0.1,35.9s0,23.9,0,35.9s-0.1,23.9-0.1,35.9s0.1,23.9,0.1,35.9
		c0,12,0.1,23.9,0.1,35.9s0,23.9,0,35.9c0,12,0,23.9,0,35.9c0,1.3,0.1,2.6,0.1,3.9c0.1,4.7-0.6,7.1,0.6,7.9c0.5,0.3,1.1,0,2.3-0.2
		c2.1-0.3,24.8-16.1,31.1-21.6c2.9-2.5,8.9-7,21-15.9c3.8-2.8,16.9-12.3,30.2-22.1c8.6-6.4,13-10.8,16.7-14.6c2.2-2.2,4.1-4.7,6.1-7
		c3.3-3.9,5-4,9.4-1.6c3.8,2.1,7.8,3.7,11.9,5.1c9.3,3.2,18.5,6.6,27.6,10.2c15.2,5.9,31.2,8.4,47.4,7.5c11.3-0.7,22.5-2,33.5-4
		c11-2,21.9-4.9,32.5-8.9c9.1-3.4,17.3-8.3,25.8-13c5.2-2.9,10.5-5.8,15.7-8.9c7.2-4.2,19.6-12.7,31.5-21.2
		c14.1-10,27.5-19.8,30.7-22.4c7.7-4.4,16.3-10.9,25.5-14.9c18.5-8.1,31-15.5,40.5-17.7c10.6-6.9,20.4-14.7,28.7-24.6
		c9.2-10.9,19.6-20.8,30.8-29.8c7.7-6.3,13.1-14.7,18.1-23.2c8.4-14.5,13.8-30.2,17.7-46.5c1.4-6,1-12.1,0.6-18.2
		c-0.5-7,0.6-13.8,3.1-20.3c3.9-10.2,5-20.8,5.5-31.6c0.3-5.9,1-11.9,2.2-17.7c1.3-6.3,4.5-11.5,8.8-16.2
		c10.3-11.2,19.7-23.1,29-35.2c3.7-4.8,7-9.9,8.9-15.7c4.3-13,11.6-24.2,20.4-34.5c6-7.1,11.8-14.4,17.8-21.5
		c11.4-13.5,20.1-28.6,26.9-44.9c4.8-11.4,10.4-22.4,18.3-32.2c4.3-5.4,7.2-11.6,8.9-18.3c0.3-1.3,0.7-2.7,1-4c0-2.7,0-5.5,0-8.2
		C690,243.7,684.5,239.6,677.9,236.7z M178.3,461.6c-1.3,5.4-5.3,9.4-8.7,13.7c-1.8,1.9-3.9,3.6-6,5.1c-6.4,4.5-14.1,7.1-22.6,6.6
		c-13.9-1-26.9-8-36.2-18.1C95.5,458.6,90,445,91,431c1-8,4-16,11-22c3.5-2.5,7-4.2,10.7-5.3c7.3-1.7,15.8-1.8,20.2,4.3
		c11.6,15.9-3.6,20.3-3.6,32.3c-1,7,3.7,15.7,9.7,10.7c7-7-2-18.5,14-24.1c17.4-6,23,16,26.2,28.2
		C179.7,457.4,179.3,459.6,178.3,461.6z M426.9,539.4c-1,11.4-6.3,21.9-14.3,30c-7.8,7.9-18.2,13.5-29.6,15.4
		c-16,2-33-2.9-43.6-15.9c-1.8-2-3.4-4.2-4.8-6.5s-2.6-4.8-3.6-7.4c-3-9-1-17,4-25c1.7-2.5,3.8-4.9,6.1-7.1c2.4-2.2,5-4.1,7.7-5.8
		c8.2-6.1,29.8-11.3,29.8,2.7c1,10-13.6,19.3-9.6,30.3c2,4,2.3,2.1,7.3,1.1c4-1,6-5,7-9c3-11-1.8-26.4,17.8-28.2
		c13.3-1.2,20,8.1,25.8,18.2C427.3,534.5,427.4,537,426.9,539.4z\\"/>
</g>
<path d=\\"M360.7,736.6\\"/>
<path class=\\"st0\\" d=\\"M291.6,696.3c2.6,2.3,86.2,41.9,129.5,103.7\\"/>
<path class=\\"st0\\" d=\\"M299.2,704.9c1.7,3,67.7,67.9,88.4,140.4\\"/>
<path class=\\"st0\\" d=\\"M308.8,706.9c3,1.9,91.7,28,144,82.3\\"/>
<path class=\\"st0\\" d=\\"M304.4,701c2.1,2.8,75.4,59.2,104.6,128.7\\"/>
</svg>",
        }
      }
    />
    <p
      className="bubble"
      onClick={[Function]}
      style={
        Object {
          "width": "7em",
        }
      }
    >
      Click my speech bubble to open the demo app.
    </p>
  </div>,
]
`;

exports[`'should render docList correctly' 1`] = `
Array [
  <div
    className="center-content"
  >
    <div
      className="splash"
    >
      <span
        className="SVGInline logo"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<svg class=\\"SVGInline-svg logo-svg\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 481.91 121.64\\"><path d=\\"M450.34 70.31l29.61-34.76h-31.56l-19.84 24.68-2.66 3.52V0h-26.33v120.08h26.33V96.02l7.42-7.43 18.44 31.49h30.16l-31.57-49.77zM368.75.08h26.41v120h-26.41zM364.81 63.52q-.23-13.75-9.61-21.65t-25.55-7.89q-16.17 0-26.6 7.65t-10.43 19.9h26.33q0-9.26 9.38-9.26 10.08 0 10.08 11.7v4.48h-8.12q-19.38 0-29.53 7T290.6 96.23a23 23 0 0 0 8.35 18.14 30 30 0 0 0 20.31 7.23q13 0 20.31-9.14a29.69 29.69 0 0 0 2.5 7.58h26.33v-1.33q-3.44-6.17-3.59-17.81V63.52zm-26.41 32.5a13.23 13.23 0 0 1-4.65 4.77 14.69 14.69 0 0 1-8.16 2.11 8.76 8.76 0 0 1-6.17-2.23 7.53 7.53 0 0 1-2.42-5.82q0-12.34 13.75-12.34h7.66v13.52z\\"/><path d=\\"M271.81 35.55l-8.51 49.82-13.28-49.84h-18.36l-13.44 51-8.36-51h-31.83v-21h-6.59c1.31 3.11-5.47 7.1-6.24 9.71-1.47 4.92 2.32 11 1.15 16.33-1.41 6.42-5.85 10.52-11 14.52-.89.7-2.26 1.67-3.69 2.8v18.67c1.36 4.54.92 7.53 5.25 11.43 2.24 2 6 3.84 7.88 5.88a22.93 22.93 0 0 1 3.4 4.64c2.37 5.2 7.2 12.79 6.64 18 .06-.53 0 .13 0 0-.29 3.21-3.79 5.49-6.39 3-1.41-1.36-2-6.24-3.24-8-5.13-7.52-9.18-10.18-13.32-11.49.61 6.58 2.69 11.67 6.27 15.21q6.48 6.41 20.08 6.41a48.61 48.61 0 0 0 14.88-2.27v-18.82a38.63 38.63 0 0 1-6.56.47q-5.08 0-6.8-1.87t-1.72-6.64V53.83h10.9l15.61 66.25h22.27l14.06-50.16 14 50.16h22.27l19.92-84.53h-25.25z\\"/><path d=\\"M158.67 37.82c-.24-6.51-3.56-12.68.53-18.95a18.38 18.38 0 0 1 4.51-4.34h-12v21h-10.79v13.08a26.07 26.07 0 0 0-5.72-6.74q-9.37-7.89-25.55-7.89t-26.6 7.65q-7.6 5.57-9.66 13.59a29.4 29.4 0 0 0-7.75-12.1q-9.69-9.14-26.17-9.14-12.19 0-21.09 5.27A34.41 34.41 0 0 0 4.73 54.37 51.48 51.48 0 0 0 0 76.99v1.09q0 20.47 10.63 32a34 34 0 0 0 7.65 6.24c1.17-1.06 3.55-1.31 4.94-2.06 2.56-1.36 4.74-3.85 6.91-5.69 4.26-3.63 10.12-7.41 12.36-12.5.72-1.63 3.83-10.57 2.72-12.46-2-3.48-8.2-1.59-10.52-1.64-3.87-.08-6.76-.39-9.49-2.72a2 2 0 0 1-.65-1.94c.67-3.15 1.52-7 3.69-9.49 1-1.12 2.84-1.92 3.28-3.38.51-1.66-.94-3.07-1-4.72a.19.19 0 0 1 .28-.16c.7.41 2.26 1.16 3.81 2-6.27-7.84 7.59.45 7.85.5 5.46 1 12.9 2.19 18.37 1.14 3.12-.6 5.73-2 9-2.05 3.71-.05 7.29 1.3 11 1 6.92-.49 13.7-2.72 20.69-3.23 7.55-.55 14.71.58 22.22.83 7.35.25 13.15-3.45 19.15-7a65.83 65.83 0 0 1 6.77-3.56c5.16-2.21 9.22-5.67 9.01-11.37zM75.02 100.06c-5-3.21-7.4-2.07-12.38-.82-2.24.56-4.53.61-6.65 1.55-4.42 2-7.62 5.86-11.85 8.16a105.74 105.74 0 0 0-12.67 8.19 29.86 29.86 0 0 1-4.81 2.78 46.9 46.9 0 0 0 13 1.72q15.7 0 25.66-8.91a29.76 29.76 0 0 0 6.57-8.3 23.52 23.52 0 0 0 7 10 29.64 29.64 0 0 0 3.85 2.8 1.4 1.4 0 0 0 .78-.6c.23-.6-.92-2.94-1-3.62-.45-4.9-3.19-10.16-7.5-12.95zM144.8 98.47a41.22 41.22 0 0 1-4.81-1.22c-1 3.11-.65 4.93.44 7.84.7 1.86 2.23 3 1.71 5.12-.76 3.16-5.38 7.19-9.81 9.86h16.06v-1.33q-3.44-6.17-3.59-17.81v-2.46zM120.65 101.76c-8.29-6.43-22.64-4-32.57-6.45-2 6.72 6.38 18.89 1.6 25a33.38 33.38 0 0 0 9.56 1.35q13 0 20.31-9.14a40.36 40.36 0 0 0 1.49 5.3 5.51 5.51 0 0 0 1-.3 20.29 20.29 0 0 0 5.92-4.73c2.84-4.04-3.85-8.34-7.31-11.03z\\"/></svg>",
          }
        }
      />
      <h3>
        Sets the stage for your Qlik data models
      </h3>
      <div>
        <p>
          WebSocket connected, but no open app. Choose one below:
        </p>
        <ul
          className="doc-list"
        >
          <li
            onClick={[Function]}
          >
            <i
              className="icon"
            />
            <span
              className="title"
            >
              <b>
                Consumer_Sales
              </b>
               
              (
              See through the eyes of a consumer goods company.  Analyze sales data by sales rep, by region, by product, etc.
              )
            </span>
          </li>
          <li
            onClick={[Function]}
          >
            <i
              className="icon"
            />
            <span
              className="title"
            >
              <b>
                FDA - Drug Cases
              </b>
               
              (
              This Dashboard is based on the 2015 Q4 data.
              )
            </span>
          </li>
          <li
            onClick={[Function]}
          >
            <i
              className="icon"
            />
            <span
              className="title"
            >
              <b>
                New_QVNCycles_with_measures_dimensions_visualizations_
              </b>
               
              (
              An app with some measures and dimensions and master visualizations.
Texten finns också på skånska.
              )
            </span>
          </li>
        </ul>
      </div>
    </div>
  </div>,
  <div
    className="cat-with-bubble"
    style={null}
  >
    <span
      className="SVGInline close-me"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg class=\\"SVGInline-svg close-me-svg\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\"><g data-name=\\"Layer 2\\"><g data-name=\\"close\\"><rect width=\\"24\\" height=\\"24\\" transform=\\"rotate(180 12 12)\\" opacity=\\"0\\"/><path d=\\"M13.41 12l4.3-4.29a1 1 0 1 0-1.42-1.42L12 10.59l-4.29-4.3a1 1 0 0 0-1.42 1.42l4.3 4.29-4.3 4.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l4.29-4.3 4.29 4.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z\\"/></g></g></svg>",
        }
      }
      onClick={[Function]}
      title="Hide cat"
    />
    <span
      className="SVGInline cat"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg class=\\"SVGInline-svg cat-svg\\" version=\\"1.1\\" id=\\"Layer_1\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" x=\\"0px\\" y=\\"0px\\"
	 viewBox=\\"0 0 864 864\\" style=\\"enable-background:new 0 0 864 864;\\" xml:space=\\"preserve\\">
<style type=\\"text/css\\">
	.st0{fill:none;stroke:#000000;stroke-width:2;stroke-linecap:round;stroke-miterlimit:10;}
</style>
<g>
	<path d=\\"M677.9,236.7c-5.9-2.7-12.3-3.5-18.7-4.3c-8.4-1-16.8-1.4-25.2-0.9c-14.9,0.8-29.5,3.5-43.9,7.2s-28.8,7.9-42.7,13.3
		c-10.8,4.2-22.2,6.1-33.5,8.2c-11.1,2.1-22.3,3.2-33.4,4.4c-11.4,1.1-22.5-0.3-32.5-6.4c-2.7-1.6-5.6-2.8-8.7-3.7
		c-11.3-3.2-21.4-8.8-31.6-14.2c-12.4-6.5-24.7-13.2-37.2-19.5c-12.5-6.4-25.1-12.5-37.9-18.3c-12.5-5.7-23.4-13.9-33.9-22.7
		c-7.5-6.4-14.9-12.7-23.4-17.7c-9.5-5.6-19.6-9.3-30.6-10.8c-7.8-1.1-13.9-5.2-18.6-11.6c-6.5-8.9-11.5-18.6-15.6-28.8
		c-4.4-10.9-8.5-22-14.1-32.4c-7.4-13.8-15.7-27-26.7-38.5c-8.8-8.9-19.2-15.2-31.4-18.2c-3.3-0.8-6.6-1.8-9.9-2.7
		c-2.7,0-5.5,0-8.2,0c-4.2,1.3-7.5,4-10.3,7.3c-2.9,3.4-6.5,6.2-9.3,9.6c-1.8,2.2-3.3,4.5-4.4,7C90.3,57,82.1,69.8,73,82.2
		c-5.5,7.6-10.3,15.5-14.6,23.7c-4.3,8.2-7.9,16.7-11.3,25.4c-2.3,6-2.8,12.5-4,18.7c-0.9,4.3-2.4,8.3-4.4,12.2
		c-2.7,5.3-4.4,11-7.4,16.2c-3.7,6.2-7.6,12.1-12.8,17.1c-1.7,1.5-13.7,10.7-13.7,11.5c0,12,0,23.9,0,35.9c0,12,0,23.9,0,35.9
		c0,12-0.1,23.9-0.1,35.9s0,23.9,0,35.9s0.1,23.9,0.1,35.9c0,12,0,23.9,0,35.9s-0.1,23.9-0.1,35.9s0,23.9,0,35.9c0,12,0,23.9,0,35.9
		s0,23.9,0,35.9c0,12,0.1,23.9,0.1,35.9s-0.1,23.9-0.1,35.9s0,23.9,0,35.9s-0.1,23.9-0.1,35.9s0.1,23.9,0.1,35.9
		c0,12,0.1,23.9,0.1,35.9s0,23.9,0,35.9c0,12,0,23.9,0,35.9c0,1.3,0.1,2.6,0.1,3.9c0.1,4.7-0.6,7.1,0.6,7.9c0.5,0.3,1.1,0,2.3-0.2
		c2.1-0.3,24.8-16.1,31.1-21.6c2.9-2.5,8.9-7,21-15.9c3.8-2.8,16.9-12.3,30.2-22.1c8.6-6.4,13-10.8,16.7-14.6c2.2-2.2,4.1-4.7,6.1-7
		c3.3-3.9,5-4,9.4-1.6c3.8,2.1,7.8,3.7,11.9,5.1c9.3,3.2,18.5,6.6,27.6,10.2c15.2,5.9,31.2,8.4,47.4,7.5c11.3-0.7,22.5-2,33.5-4
		c11-2,21.9-4.9,32.5-8.9c9.1-3.4,17.3-8.3,25.8-13c5.2-2.9,10.5-5.8,15.7-8.9c7.2-4.2,19.6-12.7,31.5-21.2
		c14.1-10,27.5-19.8,30.7-22.4c7.7-4.4,16.3-10.9,25.5-14.9c18.5-8.1,31-15.5,40.5-17.7c10.6-6.9,20.4-14.7,28.7-24.6
		c9.2-10.9,19.6-20.8,30.8-29.8c7.7-6.3,13.1-14.7,18.1-23.2c8.4-14.5,13.8-30.2,17.7-46.5c1.4-6,1-12.1,0.6-18.2
		c-0.5-7,0.6-13.8,3.1-20.3c3.9-10.2,5-20.8,5.5-31.6c0.3-5.9,1-11.9,2.2-17.7c1.3-6.3,4.5-11.5,8.8-16.2
		c10.3-11.2,19.7-23.1,29-35.2c3.7-4.8,7-9.9,8.9-15.7c4.3-13,11.6-24.2,20.4-34.5c6-7.1,11.8-14.4,17.8-21.5
		c11.4-13.5,20.1-28.6,26.9-44.9c4.8-11.4,10.4-22.4,18.3-32.2c4.3-5.4,7.2-11.6,8.9-18.3c0.3-1.3,0.7-2.7,1-4c0-2.7,0-5.5,0-8.2
		C690,243.7,684.5,239.6,677.9,236.7z M178.3,461.6c-1.3,5.4-5.3,9.4-8.7,13.7c-1.8,1.9-3.9,3.6-6,5.1c-6.4,4.5-14.1,7.1-22.6,6.6
		c-13.9-1-26.9-8-36.2-18.1C95.5,458.6,90,445,91,431c1-8,4-16,11-22c3.5-2.5,7-4.2,10.7-5.3c7.3-1.7,15.8-1.8,20.2,4.3
		c11.6,15.9-3.6,20.3-3.6,32.3c-1,7,3.7,15.7,9.7,10.7c7-7-2-18.5,14-24.1c17.4-6,23,16,26.2,28.2
		C179.7,457.4,179.3,459.6,178.3,461.6z M426.9,539.4c-1,11.4-6.3,21.9-14.3,30c-7.8,7.9-18.2,13.5-29.6,15.4
		c-16,2-33-2.9-43.6-15.9c-1.8-2-3.4-4.2-4.8-6.5s-2.6-4.8-3.6-7.4c-3-9-1-17,4-25c1.7-2.5,3.8-4.9,6.1-7.1c2.4-2.2,5-4.1,7.7-5.8
		c8.2-6.1,29.8-11.3,29.8,2.7c1,10-13.6,19.3-9.6,30.3c2,4,2.3,2.1,7.3,1.1c4-1,6-5,7-9c3-11-1.8-26.4,17.8-28.2
		c13.3-1.2,20,8.1,25.8,18.2C427.3,534.5,427.4,537,426.9,539.4z\\"/>
</g>
<path d=\\"M360.7,736.6\\"/>
<path class=\\"st0\\" d=\\"M291.6,696.3c2.6,2.3,86.2,41.9,129.5,103.7\\"/>
<path class=\\"st0\\" d=\\"M299.2,704.9c1.7,3,67.7,67.9,88.4,140.4\\"/>
<path class=\\"st0\\" d=\\"M308.8,706.9c3,1.9,91.7,28,144,82.3\\"/>
<path class=\\"st0\\" d=\\"M304.4,701c2.1,2.8,75.4,59.2,104.6,128.7\\"/>
</svg>",
        }
      }
    />
    <p
      className="bubble"
      onClick={[Function]}
      style={
        Object {
          "width": "7em",
        }
      }
    >
      Click my speech bubble to open the demo app.
    </p>
  </div>,
]
`;
